internal/datastore/common/helpers.go

```diff
@@ -0,0 +1,26 @@
+package common
+
+import (
+	"context"
+
+	"github.com/authzed/spicedb/pkg/datastore"
+	core "github.com/authzed/spicedb/pkg/proto/core/v1"
+)
+
+func WriteTuples(ctx context.Context, ds datastore.Datastore, op core.RelationTupleUpdate_Operation, tuples ...*core.RelationTuple) (datastore.Revision, error) {
```

Comment:
Technically I think this should be `UpdateTuplesInDatastore`, because the update op is given